bee-message/src/output/native_token.rs

```diff
@@ -40,12 +44,21 @@ impl NativeToken {
     }
 }
 
+#[inline]
```

Comment:
Are we sure about the explicit inlining. I still hear Josh screaming about such things :trollface:

Reply:
The function is needed to avoid passing a closure to packable but realistically we want to avoid the call if possible. Also this is just `inline`, not `inline(always)` so the compiler will do whatever it wants

Reply:
yea, I noticed that...I am wondering now if there's a real difference between not adding the attribute and adding `inline` only? :thinking:

Reply:
Ask Christian :trollface:

Reply:
I've seen cases where the compiler doesn't inline short functions (just returning a field or calling a method of a field). Given that the function is only used inside the packable implementation I think `#[inline]` puts us in the safe side.